### PR TITLE
HTML structure prepared

### PR DESCRIPTION
**Pull request by Geril**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    padding-bottom: 75px;
+    font-weight: bold;
+    color: #d0a610;
+}
+.current-roll {
+    display: block;
+    font-size: 20px;
+    padding-bottom: 15px;
+}
+button {
```
Comment (hayden7913) [inline css/style.css]: Use two spaces between style blocks

File: css/style.css [hunk 2]
```diff
@@ -8,6 +8,7 @@ body {
 #wrapper {
     box-sizing: border-box;
     padding: 25px;
+    width: 95%;
     background: rgba(31, 31, 31, 0.54);
```
Comment (hayden7913) [inline css/style.css]: Please use blue for a background color

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    background-size: cover;
+    font-family: "Raleway", sans-serif;
+    color: #fff;
+}
+/* d0a610 */
+#wrapper {
+    box-sizing: border-box;
+    padding: 25px;
+    width: 95%;
+    background: rgba(31, 31, 31, 0.54);
```
Comment (hayden7913) [inline css/style.css]: Use blue for background

Comment (Geril) [inline css/style.css]: I don't think this is an good idea, I won't look good using this BG.

Comment (hayden7913) [inline css/style.css]: Please use two space between style blocks

Comment (hayden7913) [inline css/style.css]: Okay, fair enough.

Comment (Geril) [inline css/style.css]: ok, cool

**Comment by Geril** [comment]
Added spaces between all CSS selectors.

File: css/style.css (hunk 3 above)
Comment (Geril) [inline css/style.css]: updated as written in comment below ( https://github.com/Geril/session-work/pull/1#issuecomment-272257470 )

**Pull request merged**